src/Fmt_ast.ml

```diff
@@ -176,15 +176,21 @@ let parens_or_begin_end c ~loc =
       let str = String.lstrip (Source.string_at c.source loc) in
       if String.is_prefix ~prefix:"begin" str then `Begin_end else `Parens
 
-let wrap_fits_breaks_exp_if ?(space = true) c ~parens ~loc k =
-  match parens_or_begin_end c ~loc with
-  | `Parens -> wrap_fits_breaks_if ~space c.conf parens "(" ")" k
-  | `Begin_end -> wrap_fits_breaks_exp_begin_end ~parens k
+let wrap_fits_breaks_exp_if ?(space = true) ?(preserve_kw = true)
```

Comment:
`preserve_kw` is never used. Can be removed.

Reply:
Fixed.